conda_smithy/lint_recipe.py

```diff
@@ -36,7 +35,12 @@ def lintify(meta):
 
     # 4: The recipe should have some tests.
     if 'test' not in major_sections:
-        lints.append('The recipe must have some tests.')
+        test_files = ['run_test.py', 'run_test.sh', 'run_test.bat', 'run_test.pl']
```

Comment:
I'll defer to @groutr and @mingwandroid on this.

On Wed, Mar 23, 2016, 06:04 Filipe <notifications@github.com> wrote:

> In conda_smithy/lint_recipe.py
> <https://github.com/conda-forge/conda-smithy/pull/90#discussion_r57141418>
> :
>
> > @@ -36,7 +35,12 @@ def lintify(meta):
> >
> >      # 4: The recipe should have some tests.
> >      if 'test' not in major_sections:
> > -        lints.append('The recipe must have some tests.')
> > +        test_files = ['run_test.py', 'run_test.sh', 'run_test.bat', 'run_test.pl']
>
> I am not sure if there is such a thing. I was actually asking.
>
> @msarahan <https://github.com/msarahan> is there a run_test for R?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/conda-smithy/pull/90/files/228ae9a276beb4d92cd5029fe14f485acf9978d8#r57141418>
>